### PR TITLE
Add session-jdbc sample

### PR DESCRIPTION
**Pull request by marcusdacoregio**
Closes gh-27

### CODE REVIEW

File: session-jdbc-security/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Tests the session persistence and cookie customization from Spring Session together with Spring Session JDBC and Spring Security.
```
Comment (mhalbritter) [inline session-jdbc-security/README.md:1]: I don't quite get why this smoke test has Security in it. Is there something in session / security which has to be tested in combination? If not, I would be in favor of removing security from that smoke test.

File: session-jdbc-security/src/main/java/com/example/session/MainController.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class MainController {
+
+	@GetMapping("/")
+	public String index() {
+		return "home";
```
Comment (mhalbritter) [inline session-jdbc-security/src/main/java/com/example/session/MainController.java:11]: If we're not needing Security, you can remove that controller.

**Review by mhalbritter — CHANGES_REQUESTED** [review]

**Comment by mhalbritter** [comment]
Hey @marcusdacoregio thanks for the sample. I have added some remarks, please take a look at them.

**Comment by marcusdacoregio** [comment]
Hi @mhalbritter, thanks for the review. The PR is now updated.

**Comment by mhalbritter** [comment]
Merged, thanks a lot @marcusdacoregio !

**Pull request merged**